pax/plugins/io/WaveformSimulator.py

```diff
@@ -133,10 +149,30 @@ def s1(self, photons, recoil_type, g4_id=-1, t=0., x=0., y=0., z=0.):
         photon_times = self.simulator.s1_photons(photons, recoil_type, x, y, z, t)
         if not len(photon_times):
             return None
-        self.store_true_peak('s1', g4_id, t, x, y, z, photon_times)
-        return self.simulator.queue_signal(photon_times, x=x, y=y, z=z)
-
-    def s2_after_pulses(self):
+        arriving_photon_times_top, arriving_photon_times_bottom =  self.simulator.queue_signal(
+                                                                                               photon_times,
+                                                                                               x=x,
+                                                                                               y=y,
+                                                                                               z=z
+                                                                                              )
+        peak_top_fraction = float(len(arriving_photon_times_top)) / float(
+            len(arriving_photon_times_bottom) + len(arriving_photon_times_top) )
+        self.store_true_peak('s1',
+                                         g4_id,
+                                         t,
+                                         x,
+                                         y,
+                                         z,
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->